### DISCUSSION TITLE
Dropdown and toggle button

### DISCUSSION THREAD

**Discussion by Vukichinuky** [heroui-inc/heroui#1024] [Feedback]
i spend at least 5h trying to combine dropdown and toggle button with no luck.. can someone help

**Comment by Vukichinuky** [comment]
import { Navbar, Dropdown, Button, Link, Text } from "@nextui-org/react";

export default function Navproba() {
    const collapseItems = [
    "О школи",
    "Новости",
    "Галерија",
    "Настава",
    "Ученици",
   
  ];
  const dropDownItems = [
    "Основни подаци",
    "Мисија и визија школе",
    "О Вуку Караџићу",
    "Савјет родитеља",
    "Школски одбор",
    "Постигнућа",
   
  ];
  return (
      <Navbar isBordered variant="sticky">
         <Navbar.Toggle className=" sm:hidden" aria-label="toggle navigation " />
        <Navbar.Content
          enableCursorHighlight
          activeColor="secondary"
          hideIn="xs"
          variant="underline"
        >
          <Dropdown isBordered>
            <Navbar.Item>
              <Dropdown.Button
                auto
                light
                css={{
                  px: 0,
                  dflex: "center",
                  svg: { pe: "none" },
                }}
                ripple={false}
              >
О школи              </Dropdown.Button>
            </Navbar.Item>
            <Dropdown.Menu
              aria-label="ACME features"
              css={{
                $$dropdownMenuWidth: "340px",
                $$dropdownItemHeight: "70px",
                "& .nextui-dropdown-item": {
                  py: "$4",
                  // dropdown item left icon
                  svg: {
                    color: "$secondary",
                    mr: "$4",
                  },
                  // dropdown item title
                  "& .nextui-dropdown-item-content": {
                    w: "100%",
                    fontWeight: "$semibold",
                  },
                },
              }}
            >
              <Dropdown.Item
                key="autoscaling"
                showFullDescription
                description="Kад је настала школа"
              >
Основни подаци              </Dropdown.Item>
              <Dropdown.Item
                key="usage_metrics"
                showFullDescription
                description=""
              >
Мисија и визија школе              </Dropdown.Item>
              <Dropdown.Item
                key="production_ready"
                showFullDescription
                description="
                Ускоро"
              >
О Вуку Караџићу              </Dropdown.Item>
              <Dropdown.Item
                key="99_uptime"
                showFullDescription
                description="
                Ускоро"
              >
Савјет родитеља              </Dropdown.Item>
              <Dropdown.Item
                key="supreme_support"
                showFullDescription
                description="
                Ускоро"
              >
Школски одбор              </Dropdown.Item>
<Dropdown.Item
                key="supreme_support"
                showFullDescription
                description="
                Ускоро."
              >
Постигнућа</Dropdown.Item>
            </Dropdown.Menu>
          </Dropdown>
          <Navbar.Link isActive href="#">
          Новости
          </Navbar.Link>
          <Navbar.Link href="http://localhost:3000/">Галерија</Navbar.Link>
          <Navbar.Link href="#">Настава</Navbar.Link>
          <Navbar.Link href="#">Ученици</Navbar.Link>

        </Navbar.Content>
        <Navbar.Content>
         
         
        </Navbar.Content>
        <Navbar.Collapse>
        {collapseItems.map((item, index) => (
          <Navbar.CollapseItem key={item}>
             <Dropdown isBordered>
            
              <Dropdown.Button
                auto
                light
                css={{
                  px: 0,
                  dflex: "center",
                  svg: { pe: "none" },
                }}
                ripple={false}
              >
{item}              </Dropdown.Button>
<Dropdown.Menu>
    
    {collapseItems.map((item, index) => (
        <Dropdown.Item> key={item}>
            <Link>{item}</Link>
    </Dropdown.Item>
</Dropdown.Menu>
</Dropdown>
          </Navbar.CollapseItem>
        ))}
      </Navbar.Collapse>
      </Navbar>
  );
}

**Comment by jrgarciadev** [comment]
Hey guys sorry for the delay please upgrade to [V2](https://nextui.org/docs/guide/upgrade-to-v2) and test this again.
